sdks/python/apache_beam/io/filebasedsource.py

```diff
@@ -189,7 +203,11 @@ def split(
         stop_position=stop_position)
 
   def estimate_size(self):
-    file_names = [f for f in fileio.ChannelFactory.glob(self._pattern)]
+    if not self._pattern.is_accessible():
+      raise RuntimeError('value not accessible')
```

Comment:
We catch, log, and ignore all exceptions raised when estimating size of sources at job submission. Also, users can disable validation when creating file-based sources if needed. So I think raising errors at these places is fine.